dev-games/ogre/ogre-2.1-r20180724.ebuild

```diff
@@ -0,0 +1,157 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils cmake-utils mercurial
+
+DESCRIPTION="Object-oriented Graphics Rendering Engine"
+HOMEPAGE="http://www.ogre3d.org/"
```

Comment:
this is already `https` in the older versions, please diff it for other outdated elements

Reply:
fixed